Infoblox/ddi/tests/query_log_dns_6.json

```diff
@@ -0,0 +1,34 @@
+{
+  "input": {
+    "message": "client 192.168.1.1#40432: UDP: query: test.test.org IN A response: NOERROR + test.test.org. 365 IN A 192.168.1.1; test.test.org. 365 IN A 192.168.1.1; test.test.org. 365 IN A 192.168.1.1; test.test.org. 365 IN A 192.168.1.1; test.test.org. 365 IN A 192.168.1.1; test.test.org. 365 IN A 192.168.1.1"
+  },
+  "expected": {
+    "message": "client 192.168.1.1#40432: UDP: query: test.test.org IN A response: NOERROR + test.test.org. 365 IN A 192.168.1.1; test.test.org. 365 IN A 192.168.1.1; test.test.org. 365 IN A 192.168.1.1; test.test.org. 365 IN A 192.168.1.1; test.test.org. 365 IN A 192.168.1.1; test.test.org. 365 IN A 192.168.1.1",
```

Comment:
Same, I think that the second part of the dns reponse can land into `dns.answers` (like in [EfficientIP solidserver DDI](https://github.com/SEKOIA-IO/intake-formats/blob/main/EfficientIP/solidserver-ddi/ingest/parser.yml#L144-L151))